src/shared/keyboard-navigation.ts

```diff
@@ -45,24 +59,15 @@ function isElementVisible(element) {
  *
  * [1] https://www.w3.org/TR/wai-aria-practices/#kbd_roving_tabindex
  * [2] https://www.w3.org/TR/wai-aria-practices/#keyboard
- *
- * @param {import('preact').RefObject<HTMLElement>} containerRef
- * @param {object} options
- *   @param {boolean} [options.autofocus] - Whether to focus the first element
- *     in the set of matching elements when the component is mounted
- *   @param {boolean} [options.horizontal] - Enable navigating elements using left/right arrow keys
- *   @param {boolean} [options.vertical] - Enable navigating elements using up/down arrow keys
- *   @param {string} [options.selector] - CSS selector which specifies the
- *     elements that navigation moves between
  */
 export function useArrowKeyNavigation(
```

Comment:
Just as an FYI, we have another version of this hook now in `@hypothesis/frontend-shared`.